README.md

```diff
@@ -92,230 +110,192 @@
-        <li><a href="#-collaborate-with-zenml">Collaborate with ZenML</a></li>
+        <li><a href="#-run-a-pipeline">Run a pipeline</a></li>
+        <li><a href="#-start-the-local-dashboard">Start the local Dashboard</a></li>
         <li><a href="#-zenbytes">ZenBytes</a></li>
         <li><a href="#-zenml-projects">ZenML Projects</a></li>
       </ul>
     </li>
     <li>
-      <a href="#%EF%B8%8F-infrastructure-requirements">Infrastructure Requirements</a>
+      <a href="#-infrastructure-requirements">Infrastructure Requirements</a>
       <ul>
         <li><a href="#-deploy-zenml">Deploy ZenML</a></li>
-        <li><a href="#-open-source-mlops-stack-recipes">MLOps Stack Recipes</a></li>
+        <li><a href="#-deploy-stack-components">Deploy Stack Components</a></li>
       </ul>
     </li>
     <li><a href="#-roadmap">Roadmap</a></li>
     <li><a href="#-contributing-and-community">Contributing and Community</a></li>
-    <li><a href="#-meet-the-team">Meet the Team</a></li>
     <li><a href="#-getting-help">Getting Help</a></li>
     <li><a href="#-license">License</a></li>
-    <li><a href="#-learn-more">Learn More</a></li>
   </ol>
 </details>
 
 <br />
 
-# 🤖 Why ZenML?
-
-🤹 Are you an ML engineer or data scientist shipping models to production and juggling a plethora of tools? ZenML is here to help transition your production ML pipelines to production.
+# 🤖 Introduction
 
-![The long journey from experimentation to production.](docs/book/assets/1-pipeline-hard-reproduce.png)
+🤹 ZenML is an extensible, open-source MLOps framework for creating portable,
+production-ready machine learning pipelines. By decoupling infrastructure from
+code, ZenML enables developers across your organization to collaborate more
+effectively as they develop to production.
 
-Everyone loves to train ML models, but few talks about shipping them into production, and even fewer can do it well. At ZenML, we believe the journey from model development to production doesn't need to be long and painful.
+-💼ZenML gives data scientists the freedom to fully focus on modeling and
+  experimentation while writing code that is production-ready from the get-go.
 
-If you come from unstructured notebooks or scripts with lots of manual processes, ZenML will make the path to production easier and faster for you and your team. Using ZenML allows you to own the entire pipeline - from experimentation to production.
+- 👨‍💻 ZenML empowers ML engineers to take ownership of the entire ML lifecycle
+  end-to-end. Adopting ZenML means fewer handover points and more visibility on
+  what is happening in your organization.
 
-Read more [here](https://blog.zenml.io/why-zenml/).
+-🛫ZenML enables MLOps infrastructure experts to define, deploy, and manage
+  sophisticated production environments that are easy to share with colleagues.
 
+![The long journey from experimentation to production.](docs/book/.gitbook/assets/intro-zenml-overview.png)
```

Comment:
Doesn't this diagram seems a bit complicated to read and it's to much for the early start of the README?